jina/types/arrays/memmap.py

```diff
@@ -64,6 +65,7 @@ def __init__(self, path: str, key_length: int = 36):
         self._body_path = os.path.join(path, 'body.bin')
         self._key_length = key_length
         self._load_header_body()
+        self._embeddings_shape = None
```

Comment:
shape?

Reply:
yes numpy memmap needs a parameter `shape` to be set, so we need to save this shape. otherwise it's recover a flat array